oddbird/MetaShare
# [not-latest-commit] Notifying if QA org is in sync with Dev org

## Cute animal pic
![wiggedrat](https://user-images.githubusercontent.com/12376362/73111481-6e250780-3ed8-11ea-973d-d07e3a4f9c0e.jpg)

## Link to Trello card (if applicable)
https://trello.com/c/3H4uO9tc & https://trello.com/c/cJOAFupK

## Description

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally._

- Run migrations
- Create a Dev Org
- Create a Review Org
- Capture changes on the Dev Org
  - see https://oddbird.slack.com/archives/CH6EN9NMA/p1576536024007000

## Impacted areas of website
_Provide URLs and/or a description of where to look._

- Task-detail page, Review Org card

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![Screenshot 2020-02-06 18 45 30](https://user-images.githubusercontent.com/552316/73988577-1d1e0600-4911-11ea-87d0-072214712b77.png)

![Screenshot 2020-02-06 18 45 42](https://user-images.githubusercontent.com/552316/73988579-1db69c80-4911-11ea-8da1-1811adb27976.png)

![Screenshot 2020-02-06 18 45 54](https://user-images.githubusercontent.com/552316/73988580-1e4f3300-4911-11ea-9534-9c15737cc4f3.png)

## Reviewer tasks:
